src/containers/profile-page/store/lineups/tracks/retrieveUserTracks.ts

```diff
@@ -9,21 +9,24 @@ type RetrieveUserTracksArgs = {
   sort?: 'date' | 'plays'
   offset?: number
   limit?: number
+  getUnlisted?: boolean
```

Comment:
can we add a comment on `getUnlisted` that informs the caller that this will sign data & won't let you get anyone's unlisted tracks